### ISSUE TITLE
Provide a simple way of parsing text

### ISSUE THREAD

**Issue by sscdotopen** [stratosphere/stratosphere#430]
I'm failing to use DecimalTextIntParser...

When I see a line like ths:

```
parser.parseField(bytes, offset, numBytes, (char) 0xffff, field);
```

I feel like this:

![entjs](https://f.cloud.github.com/assets/409707/1985177/6e80d9ca-843a-11e3-869e-cf5a00e2027c.png)

**Comment by aljoscha** [comment]
No-one is supposed to be using this stuff, right?

**Comment by sscdotopen** [comment]
It is used in the example for Spargel.

**Comment by uce** [comment]
Yeah, in [DuplicateLongInputFormat](https://github.com/stratosphere/stratosphere/blob/master/stratosphere-addons/spargel/src/main/java/eu/stratosphere/spargel/java/examples/connectedcomponents/DuplicateLongInputFormat.java?source=cc).

**Comment by uce** [comment]
Btw: :+1: for providing easier to use utility classes like the parsers.

**Comment by aljoscha** [comment]
Ok, I'm not exactly familiar with this newfangled stuff.

**Comment by StephanEwen** [comment]
I think one thing we should really do is get rid of this duplicate long
format. It should really be parsing a long (could be a special case of a
CSV) and then a map function to duplicate the value.

**Comment by StephanEwen** [comment]
The parsers could also be renamed (I think binary parser is more accurate)
and they should have a method without giving an optional delimiter
